tests/unit/via/views/conftest.py

```diff
@@ -1,2 +1 @@
 # Import this so that our webargs customization is registered in the view tests.
-import via.views.webargs  # pylint: disable=unused-import
```

Comment:
FIXME. Apparently this line wasn't needed because `ruff check --fix` removed it and the tests are still passing. If removing this line need to also remove the comment above it. Or don't remove this line and add a `# noqa` instead. I'm not sure which is correct.